functions/validator/main.py

```diff
@@ -1,5 +1,59 @@
-""" This is the body of the lambda function """
+""" Mozilla Change Integration Service Validator.
+
+This lambda function is invoked by a change integration service producer like
+Mozillian's.
+
+Call:  synchronously using boto3('lambda') client.
+Takes: an event containing a user profile
+Does:  validate the user profile against the json schema for Mozilla users.
+
+If the profile passes store it in kinesis for processing.
+
+"""
+import base64
+import json
+import logging
+import os
+import utils
+
+# Import the Mozilla CIS library to facilitate core logic interaction.
+from cis import validation
+from cis import encryption
 
 def handle(event, context):
-    """This is the main handler called during function invoke."""
-    pass
+    """This is the main handler called during function invocation."""
+
+    # Initialize Stream Logger
+    # Log level can be environment driven later in development.
+    log_level = logging.INFO
+    utils.set_stream_logger(level=log_level)
+    logger = logging.getLogger('cis-validator')
+    logger.info("Validator successfully initialized.")
+
+    payload_status = validation.validate(**event)
```

Comment:
For this I actually need to run this in lambda a couple of times to see what the event looks like coming through.  It's a tiny bit different than what we have setup here I suspect.